src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(2, 2))
+    .toEqual([1, 1]);
```

Comment:
The expected result of splitInteger(2, 2) should be [1, 1], but the function is returning [1, 1]. You may want to check your test case and make sure it's correct.